N/Ncurses/build_tarballs.jl

```diff
@@ -64,6 +64,7 @@ for lib in ncurses form panel menu; do
     ln -s "${lib}w.pc" "${prefix}/lib/pkgconfig/${lib}.pc"
     ln -s "lib${lib}w.${dlext}" "${libdir}/lib${lib}.${dlext}"
 done
+ln -s ncursesw ${prefix}/include/ncurses
```

Comment:
Doesn't `ncursesw` need to be a full path here?

Reply:
A symbolic link can also be a relative path.  Since this will be untarred on users' machines that probably don't have `/workspace/destdir` I think it's better to use the relative path

Reply:
Right, so this is creating the symlink in the current directory?

Reply:
No, the symlink is created in `${prefix}/include/ncurses`, pointing to `ncursesw`, relative to where the symlink sits

Reply:
Oh, I didn't realize that's what happened when you structure `ln` calls this way. Cool.

Reply:
Actually, the auditor of BinaryBuilder transforms absolute symlinks to relative symlinks before creating the tarball, so that also having the full absolute symlink here shouldn't be a problem in the end.